Source/Core/DolphinQt2/DPI.h

```diff
@@ -0,0 +1,18 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <QPixmap>
+#include <QString>
+
+class QPaintDevice;
+
+namespace DPI
+{
+double GetScale(const QPaintDevice* device);
+double GetScaledIconSize(const double scale);
+const QString GetScaledIconSuffix(const double scale, const double max = 4);
+QPixmap GetScaledPixmap(const QPaintDevice* device, const QString& path, bool icon_scalable = true);
```

Comment:
The `bool` here should likely be replaced with an `enum class` like ScalingType or Scaling.

```cpp
enum class Scaling
{
  None,
  Scale // or w/e is more appropriate a name.
};
```

which makes it more obvious what's being requested at call sites.